modules/installation-and-upgrade/pages/container-deployment/suma/server-deployment-vm-suma.adoc

```diff
@@ -116,13 +117,13 @@ image::root-password.png[]
 
 
 
-== Register {sl-micro} and {productname} {productnumber} Server
+== Register {sle-micro} and {productname} {productnumber} Server
 
 
-.Procedure: Registering {sl-micro} and {productname} {productnumber}
+.Procedure: Registering {sle-micro} and {productname} {productnumber}
 . Boot the virtual machine.
 . Log in as `root`.
-. Register {sl-micro} with SCC.
+. Register {sle-micro} with SCC.
```

Comment:
Sorry I was too fast before. For 5.0 it's true it should be `sle-micro`. And for master currently too, but it will change rather soonish when `sl-micro` will be correct.

So for master it should remain `sl-micro`, 5.0 should have `sle-micro`